x/ibc/20-transfer/genesis.go

```diff
@@ -10,6 +10,7 @@ import (
 // InitGenesis sets distribution information for genesis
 func InitGenesis(ctx sdk.Context, keeper Keeper) {
 	// check if the module account exists
+	// TODO: should we create the module account if it doesn't exist?
```

Comment:
the desired logic is to panic the application if the module account is not initialized on `app.go`